pkg/login/social/connectors/github_oauth.go

```diff
@@ -62,13 +62,23 @@ var (
 )
 
 func NewGitHubProvider(info *social.OAuthInfo, cfg *setting.Cfg, orgRoleMapper *OrgRoleMapper, ssoSettings ssosettings.Service, features featuremgmt.FeatureToggles) *SocialGithub {
-	teamIdsSplitted := util.SplitString(info.Extra[teamIdsKey])
+	s := newSocialBase(social.GitHubProviderName, orgRoleMapper, info, features, cfg)
+
+	teamIdsSplitted, err := util.SplitStringWithError(info.Extra[teamIdsKey])
+	if err != nil {
+		s.log.Error("Invalid settings for team_ids in Github OAuth", "config", teamIdsKey, "error", err)
```

Comment:
nit: Instead of having the integration name in the message could you please add the provider as a parameter? (It would make a bit simpler to search for these) 

Something like
```
s.log.Error("Invalid settings for team_ids", "config", teamIdsKey, "provider", social.GitHubProviderName, "error", err)
```

Or to make the error message more generic:
```
s.log.Error("Invalid auth configuration setting", "config", teamIdsKey, "provider", social.GitHubProviderName, "error", err)
```

Wdyt? @volcanonoodle @dmihai